lib/utils/webpack.config.js

```diff
@@ -106,13 +106,13 @@ module.exports = (program, directory, stage, webpackPort = 1500, routes = []) =>
           }),
         ]
       default:
-        throw new Error('The state requested ' + stage + " doesn't exist.")
+        throw new Error('The state requested ' + stage + ' doesn\'t exist.')
     }
   }
 
   function resolve () {
     return {
-      extensions: ['', '.js', '.jsx', '.cjsx', '.coffee', '.json', '.less', '.toml', '.yaml'],
```

Comment:
If you're getting rid of all coffeescript (which I am whole-heartedly for), then you should also drop support for building site content using it. But I'm personally very anti-coffeescript :smile: